force-app/main/default/classes/FeatureParameters.cls

```diff
@@ -12,18 +12,28 @@ update the methods within the inner class accordingly
 
 public inherited sharing class FeatureParameters {
     @TestVisible
-     private enum DeveloperName {
+    private enum DeveloperName {
         ACTIVE_PROGRAMS,
         ACTIVE_PROGRAMS_WITH_ENGAGEMENTS_LAST30,
-        ACTIVE_PROGRAMS_WITH_SERVICE_DELIVERIES_LAST30
+        ACTIVE_PROGRAMS_WITH_SERVICE_DELIVERIES_LAST30,
+        PERM_SET_DELIVER_USERS,
+        PERM_SET_MANAGE_USERS
     }
 
     private static final String ACTIVE_PROGRAM_CONDITION =
         String.valueOf(Program__c.Status__c) +
         ' = ' +
         '\'Active\'';
-
     private static final String CREATED_LAST_30_CONDITION = 'CreatedDate = LAST_N_DAYS:30';
+    private static final String PERMISSION_SET_NAME_CONDITION =
+        String.valueOf(PermissionSet.SObjectType) +
+        '.' +
+        String.valueOf(PermissionSet.Name) +
+        ' = ';
+    @TestVisible
+    private static final String DELIVER_PERMISSION = 'PMDM_Deliver';
```

Comment:
Maybe more of a question for the note on the PR, isn't the customer-facing permission PMM: Deliver and PMM: Manage?

Reply:
Yes thanks!